release-doc-in-process/all_vf.TTL

```diff
@@ -618,6 +618,22 @@ vf:conformsTo
         vs:term_status      "unstable" ;
         rdfs:comment        "The primary resource knowledge specification or definition of an existing or potential resource." .
 
+vf:stage
+        a                   owl:ObjectProperty ;
+        rdfs:label          "stage" ;
+        rdfs:domain         [ owl:unionOf (vf:Commitment vf:RecipeFlow) ] ;
+        rdfs:range          vf:ProcessSpecification ;
+        vs:term_status      "unstable" ;
+        rdfs:comment        "The last stage the desired economic resource went through." .
+
+vf:state
+        a                   owl:ObjectProperty ;
+        rdfs:label          "state" ;
+        rdfs:domain         [ owl:unionOf (vf:Commitment vf:RecipeFlow) ] ;
```

Comment:
For RecipeFlows and Commitments to use state requirements, they need to know the states of EconomicResources, altho the information can be derived from their events.